src/main/kotlin/no/nav/syfo/papirsykemelding/model/LoggingMeta.kt

```diff
@@ -0,0 +1,8 @@
+package no.nav.syfo.papirsykemelding.model
+
+data class LoggingMeta(
+    val mottakId: String,
```

Comment:
Tror kanskje ikke vi har noen mottaksid for papirsykmeldinger (den settes av emottak for andre sykmeldinger).

Reply:
Sant, og jeg tror at den ReceivedSykemelding har et non-optional felt som er navLogId. Det er denne som er brukt for mottakId. Er det riktig?

Reply:
Det stemmer, tenker vi fyller det inn med noe fornuftig i papirsykmeldingen